ibis/file/utils.py

```diff
@@ -0,0 +1,6 @@
+try:
+    import pathlib  # noqa
```

Comment:
Can you put the specific `flake8` error code here?

Reply:
i remove this file, only used in 1 place